android/src/main/java/com/doublesymmetry/trackplayer/service/MusicService.kt

```diff
@@ -224,7 +224,7 @@ class MusicService : HeadlessJsTaskService() {
     }
 
     fun seekTo(seconds: Float) {
-        handler.post { player.seek(seconds.toLong(), TimeUnit.SECONDS) }
+        handler.post { player.seek((seconds * 1000).toLong(), TimeUnit.MILLISECONDS) }
```

Comment:
This can stay the same, no? We are still seeking only to seconds.

Reply:
Why do you want to only seek to seconds? In my use case I would like to be able to seek to milliseconds.

Reply:
Since the technical support for it is already here, it doesn't make sense to block off reasonable use cases unless there is a tradeoff I'm unaware of.

Reply:
We are fine with seeking to milliseconds. I'm just wondering if we should not take milliseconds as parameter instead of seconds as `Float`